test/e2e/support/focus-commands.js

```diff
@@ -0,0 +1,34 @@
+Cypress.Commands.add( 'auditBlockFocus', ( blockType ) => {
+	// Open up the Insert Block Menu
+	cy.get( 'button.editor-inserter__toggle:first' ).click()
```

Comment:
We should be consistent with the way we do actions across tests. Use the same selectors: I'm using `.editor-visual-editor [aria-label="Insert block"]` because I think we should try to target text and labels instead of classnames as much as we can.

I wonder if something like `cy.openInserter()` would be a good Idea?

Reply:
Let me promote using data attributes like `data-test="editor-inserter"` again. It would make tests much easier to maintain. In case of any change in the HTML code, you would move data attribute and tests should pass without any modification :)

Reply:
Sounds good to me.